.drone.yml

```diff
@@ -213,7 +213,7 @@ services:
   ports:
   - 9000
 - name: fakegcs
-  image: fsouza/fake-gcs-server
+  image: fsouza/fake-gcs-server:1.18.3
```

Comment:
👍 It's a good practice ti pinned the version. 
However, I'd love to use the latest version of it. https://github.com/fsouza/fake-gcs-server/releases/tag/v1.19.2
What's the blocker? Did they change an API? 
It'd be great if you could create an issue about it, if you have any insight?

Reply:
I really don't know anything about gcs. The only thing I know is that latest version breaks the tests, so I switched to latest version which supports tests green. I'll create an issue.

Reply:
@pastushenkoy I see. Could you then create a separate PR for this change? To keep changes hygienic. 
If not, let me know I cant do it.